### DISCUSSION TITLE
Transfer management of servers between instances

### DISCUSSION THREAD

**Discussion by danielrosehill** [coollabsio/coolify#2106] [Feature Requests]
Maybe this is something that Docker Swarm and all that is designed to handle, but it would be very useful to be able to seamlessly transfer control over a server from one Coolify instance to the other.

Server A (a management server) relinquishes control over the remote Docker host to server B which picks is up and then automatically detects the running deployments and integrates them into its own environments. 

And as it would be Coolify -> Coolify perhaps all the metadata could be just handed over directly

**Comment by reachcreator** [comment]
This would be an awesome feature and definitely one that I'd find valuable too.
